lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -971,6 +971,24 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
             rect.bottom,
           );
       }
+
+      if (rect.top < 0) {
+        final shiftAmount = rect.top;
```

Comment:
I think we should have a positive value as a `shiftAmount`, we should shift it downward.
and shift the bottom like this:
`rect.bottom + shiftAmount`
it is simpler.

see line 958:
`final shiftAmount = 0 - rect.left;`

Reply:
Actually, I got rid of shiftAmount there entirely and imho it's even simpler now:
```
rect = Rect.fromLTRB(rect.left,
          0,
          rect.right,
          tooltipHeight,
        );
```

What do you think ?

Reply:
It's great, 
Thanks!